.github/workflows/wagon-tests.yml

```
@@ -64,6 +64,56 @@ jobs:
         run: |
           bundle exec rake app:rubocop
 
+  wagon_seeds:
+    name: seeds
+    runs-on: 'ubuntu-20.04'
+    env:
+      HEADLESS: true
+      RAILS_DB_ADAPTER: postgresql
+      RAILS_DB_HOST: 127.0.0.1
+      RAILS_DB_PORT: 5432
+      RAILS_DB_USERNAME: hitobito
+      RAILS_DB_PASSWORD: hitobito
+      RAILS_DB_NAME: hitobito_test
+      RAILS_TEST_DB_NAME: hitobito_test
+      RAILS_ENV: test
```

Since we mostly have special seeds in den dev-env, we need to use that env to run them in this step.

```suggestion
      RAILS_ENV: development
```

Another option would be symlinking or copying the seeds from `db/seeds/development` to `db/seeds/test`.